src/lib/Preview.ts

```diff
@@ -58,32 +58,52 @@ export class Preview extends EventEmitter {
         resolvedPort = 2044
       }
     }
+
+    // PoC - TODO: need to be typed
+    let sceneId: string = ''
+    let sceneType: string = 'parcel'
+    const assetJsonPath = path.resolve(this.dcl.getWorkingDir(), 'asset.json')
+    if (fs.existsSync(assetJsonPath)) {
+      try {
+        const assetJson: any = require(assetJsonPath)
+        if (assetJson?.id && assetJson?.assetType === 'portable-experience'){
+          sceneId = assetJson.id
+          sceneType = 'portable-experience'
+        }
+      } catch (err) {}
```

Comment:
Always log errors, it will help us debug when we receive support tickets and will help users of the tool understand and fix the issues by themselves
```suggestion
      } catch (err) {
        console.error(err)
      }
```

Reply:
Thanks, I forgot it